modules/nextMillenniumBidAdapter.js

```diff
@@ -199,26 +199,52 @@ export const spec = {
 
   getUserSyncs: function (syncOptions, responses, gdprConsent, uspConsent) {
     const pixels = [];
-    let syncUrl = SYNC_ENDPOINT;
 
-    if (gdprConsent && gdprConsent.gdprApplies) {
-      syncUrl += 'gdpr=1&gdpr_consent=' + gdprConsent.consentString;
-    }
-    if (uspConsent) {
-      syncUrl += 'us_privacy=' + uspConsent;
-    }
+    if (isArray(responses)) {
+      responses.forEach(response => {
+        if (
+          response.body &&
+          response.body.ext &&
+          response.body.ext.sync
+        ) {
+          const sync = deepAccess(response, 'body.ext.sync', {})
+          if (syncOptions.pixelEnabled && isArray(sync.image)) {
+            sync.image.forEach(imgUrl => {
+              pixels.push({
+                type: 'image',
+                url: replaceUsersyncMacros(imgUrl, gdprConsent, uspConsent)
+              });
+            })
+          }
```

Comment:
My bad if I wasn't clear enough, the use of `deepAccess` was to avoid de `if` block.

```suggestion
  const sync = deepAccess(response, 'body.ext.sync', {})
  if (syncOptions.pixelEnabled && isArray(sync.image)) {
    sync.image.forEach(imgUrl => {
      pixels.push({
        type: 'image',
        url: replaceUsersyncMacros(imgUrl, gdprConsent, uspConsent)
      });
    })
  }
```

Reply:
@osazos Now I'm using deepAccess twice: when iframeEnabled and pixelEnabled.